paddle/math/BaseMatrix.h

```diff
@@ -457,6 +457,13 @@ class BaseMatrixT {
 
   /**
    * @code
+   * swap(this, b)
+   * @endcode
+   */
+  void deepSwap(BaseMatrixT& b);
```

Comment:
As here adds a new method, if we really think we need it, we'd write unit tests for it.